### PR TITLE
Android 3.x changes - Gradle, Github Scripts and CircleCI updates

### PR DESCRIPTION
**Pull request by PravinPK**
Other than Gradle, CircleCI and  Github action scripts. Here are the major things to look for in this PR

- Commented out a few tests in 
[MessagingFullscreenEventListenerTests.java](https://github.com/adobe/aepsdk-messaging-android/pull/239/commits/4a7058a7addb3cd5dec0497f9cf4a913facfda49#diff-2a2c59749517e933a5945db7948321a1c960d0be4ee5b9ff53cdb8c67394899c) 
[PresentableMessageMapperTests.java](https://github.com/adobe/aepsdk-messaging-android/pull/239/commits/4a7058a7addb3cd5dec0497f9cf4a913facfda49#diff-beade2f407bfe9bc2b0b00d6d92865d5025128492ef7370cafe653bb82fe5392)
[MessagingExtensionTests.java](https://github.com/adobe/aepsdk-messaging-android/pull/239/commits/4a7058a7addb3cd5dec0497f9cf4a913facfda49#diff-c5e911139100a9e8241d0e4b36f9283315696770814e4dee43119ba9dd7941ad)

- Renamed TestApp folder name

### CODE REVIEW

File: code/messagingtestutils/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+dependencies {
+    implementation(project(":messaging"))
+    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion-SNAPSHOT")
+
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.12.7")
+    implementation("androidx.test.ext:junit:1.1.5")
+
+    implementation("androidx.appcompat:appcompat:1.6.1")
+    implementation("androidx.constraintlayout:constraintlayout:2.1.4")
```
Comment (spoorthipujariadobe) [inline code/messagingtestutils/build.gradle.kts:46]: Do we use appcompat and constraintlayout in messagingtestutils? Looks like these need to be in app gradle file

Comment (PravinPK) [reply code/messagingtestutils/build.gradle.kts:46]: That slipped in.. We dont need that

**Review by spoorthipujariadobe — APPROVED** [review]
Small comment, looks good otherwise

**Comment by PravinPK** [comment]
Thanks for the quick review @spoorthipujariadobe . Will merge it after getting the build to pass.

**Pull request merged**